README.md

```diff
@@ -255,16 +254,41 @@ available any more.
 
 ### Alternative approach when using podman as the container runtime
 
-When using `podman` as the container runtime, add the following to `/etc/containers/mounts.conf` (when building containers as `root`) or `~/.config/containers/mounts.conf` (when building containers as a regular user):
-```
+When using `podman` as the container runtime, add the following to
+`/etc/containers/mounts.conf` (when building containers as `root`) or
+`~/.config/containers/mounts.conf` (when building containers as a regular
+user):
+
+```text
 <path_on_host>/SUSEConnect:/etc/SUSEConnect
 <path_on_host>/SCCcredentials:/etc/zypp/credentials.d/SCCcredentials
 ```
 
 No further change are needed in `Dockerfile`.
 
+### Obtaining the SUSEConnect and SCCcredentials secrets
+
+Ideally the host system on which your container builds are running is
+registered. In all other cases, you need to manually start the container you
+would like to register and execute `SUSEConnect` inside:
```

Comment:
```suggestion
Ideally the host system on which the container builds are executed is
registered. Otherwise start the container that should be registered 
and execute `SUSEConnect` inside it:
```